language/go/generate_test.go

```diff
@@ -90,6 +90,7 @@ func TestGenerateRules(t *testing.T) {
 			f.Sync()
 			got := string(bzl.Format(f.File))
 			wantPath := filepath.Join(dir, "BUILD.want")
+			t.Logf("Using %s file for test", wantPath)
```

Comment:
Let's drop this log statement.

Reply:
OK, but without Log, it is not always obvious on which testdata the problem occurred...

Reply:
Each subtest should have the testdata subdirectory as part of its name. Or am I missing something?